Update instructions for setting default GRUB boot menu entry (#509)
harvester/docs: docs/troubleshooting/os.md

}
```

> The id of the above entry is `debug`. We can then set the default entry by running:

Reviewer: ```suggestion
You can configure the default entry by running the following commands:  
```

```
# mount -o remount,rw /run/initramfs/cos-state
# grub2-editenv /run/initramfs/cos-state/grub_oem_env set saved_entry=debug
```

> This change can be reverted later if necessary by running `grub2-editenv /run/initramfs/cos-state/grub_oem_env unset saved_entry`

Reviewer: ```suggestion
If necessary, you can undo the change by running the command `grub2-editenv /run/initramfs/cos-state/grub_oem_env unset saved_entry`.  
```
